src/main/java/org/broadinstitute/hellbender/utils/config/GATKConfig.java

```diff
@@ -1,5 +1,6 @@
 package org.broadinstitute.hellbender.utils.config;
 
+import com.netflix.servo.util.VisibleForTesting;
```

Comment:
Its nice to think that the gatk is dependent on netflix....

Reply:
fixed, it now depends on myspace